contracts/atm/TLRToken.sol

```diff
@@ -257,7 +257,12 @@ contract TLRToken is
      * - `recipient` cannot be the zero address.
      * - the caller must have a balance of at least `amount`.
      */
-    function transfer(address recipient, uint256 amount) public returns (bool) {
+    function transfer(address recipient, uint256 amount)
+        public
+        whenNotPaused()
```

Comment:
Are we saying that if the entire platform is paused, we are not allowing users to transfer their tokens?